tasks/gateway.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Include initial OS-specific tasks
+  ansible.builtin.include_tasks: init_{{ ansible_os_family | lower }}.yml
+  vars:
+    _cvmfs_role: gateway
+    _cvmfs_upgrade: "{{ cvmfs_upgrade_server }}"
+
+- name: Create gateway API keys for repository publishing access (stratum 0)
+  template:
+    src: repos.json.j2
```

Comment:
You should test this role first. Here `src: repos.json.j2` with a `s` but the template is `templates/repo.json.j2` without the `s`.

Reply:
fixed